package.json

```diff
@@ -53,11 +53,11 @@
   },
   "scripts": {
     "compile-styles": "lessc ./node_modules/antd/dist/antd.less ./src/styles/antd.css --js",
-    "start": "yarn compile-styles && ESLINT_NO_DEV_ERRORS=true react-scripts start",
-    "build": "react-scripts build",
+    "start": "yarn compile-styles && ESLINT_NO_DEV_ERRORS=true react-app-rewired start",
+    "build": "react-app-rewired build",
```

Comment:
updated the command per the documentation: https://github.com/timarney/react-app-rewired#3-flip-the-existing-calls-to-react-scripts-in-npm-scripts-for-start-build-and-test